components/common/GlobalNav.tsx

```diff
@@ -296,6 +296,32 @@ const NavbarToggler = styled.button`
   }
 `;
 
+const CustomToolbar = (props) => {
+  const { items, mobile } = props;
```

Comment:
Please sprinkle some prop types (I haven't tested this)

```suggestion
type CustomToolbarProps = {
  items: { slug: string, id: string, icon: string }[];
  mobile?: boolean;
}

const CustomToolbar = ({ items, mobile }: CustomToolbarProps) => {
```

Reply:
Ok... Fiiiiine